src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -181,7 +311,7 @@ public void processLti(HttpServletRequest request,
 		if(appExtSecurePropertiesFile!=null) {
 			ltiKey = appExtSecurePropertiesFile.getProperty(SectionUtilityToolFilter.PROPERTY_LTI_KEY);
 			ltiSecret = appExtSecurePropertiesFile.getProperty(SectionUtilityToolFilter.PROPERTY_LTI_SECRET);
-			ltiUrl = appExtPropertiesFile.getProperty(SectionUtilityToolFilter.PROPERTY_LTI_URL);
+			ltiUrl = request.getRequestURL().toString();
```

Comment:
What changed so that it is not necessary to read the ltiUlr as a property anymore?